code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -93,6 +103,12 @@
 	playsound(src, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
 	return (BRUTELOSS)
 
+/obj/item/hatchet/wooden

```

Comment:
Wasn't anything supposed to be happening to this or did you just forget about it? No crafting recipie, no mention in changelog even with the effort to add sprites to [this pr's icons/obj/items_and_weapons.dmi](https://github.com/tgstation/tgstation/blob/6f60a8cbca2c54d7835bcec8862e9b70d91c101f/icons/obj/items_and_weapons.dmi) ?

Reply:
I added it to the ashwalker base. That's about it.

That was the original vision of this PR, not to add more improvised weapons. I think it was mentioned in the original PR description, might have removed it by accident.